save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/KotlinUtils.kt

```diff
@@ -4,4 +4,29 @@
 
 package com.saveourtool.save.utils
 
+import kotlinx.coroutines.delay
+
 typealias StringList = List<String>
+
+/**
+ * Run [action] several [times] with [timeMillis] milliseconds
+ *
+ * [T] is just a non-nullable type
+ *
+ * @param times number of times to retry [action]
+ * @param timeMillis number of milliseconds to wait until next retry
+ * @param action action that should be invoked
+ * @return [T] if the result was fetched in [times] attempts, null otherwise
+ */
+suspend fun <T : Any> retry(
+    times: Int,
+    timeMillis: Long? = 10_000L,
```

Comment:
`retry` is used to perform an action to get a result. If result is not gotten it seems to be a modern kotlin solution to return `null` instead of result of type `T`

Reply:
Thought of ignoring `duration`, but later understood that ignoring is the same as waiting 0 milliseconds 🥲

Reply:
Made it non-nullable